### PR TITLE
DCD-466: Add NAT Gateway to security & Add Export Prefix

### PR DESCRIPTION
**Pull request by nanux**
Fixes https://jira.atlassian.com/projects/SCALE/issues/SCALE-2
* Add NAT IPs to Security Group
* Add ExportPrefix parameter
* Initial support for GovCloud

### CODE REVIEW

File: templates/quickstart-confluence-master.template.yaml [hunk 1]
```diff
@@ -1066,8 +1062,8 @@ Resources:
               Service: [ec2.amazonaws.com]
             Action: ['sts:AssumeRole']
       ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM
-        - arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AmazonEC2RoleforSSM"
```
Comment (bpar476) [inline templates/quickstart-confluence-master.template.yaml:1065]: Is this a fix for govcloud?

**Review by bpar476 — APPROVED** [review]

**Review by bpar476 — CHANGES_REQUESTED** [review]
Oh noes looks like this has broken the build. They couldn't find the exported NAT IPs, we need to update the stack in the CI region.

File: templates/quickstart-confluence-master.template.yaml (hunk 1 above)
Comment (nanux) [reply templates/quickstart-confluence-master.template.yaml:1065]: Yes, the ARN is composed from the AWS partition, so in most cases it is `aws` but in govcloud it is `aws-us-gov` (and also `aws-cn` in China).

**Comment by nanux** [comment]
> Oh noes looks like this has broken the build. They couldn't find the exported NAT IPs, we need to update the stack in the CI region.

The stack was updated and running the builds now - they were failing yesterday due to intermittent issues with Artifactory.

**Review by bpar476 — APPROVED** [review]

**Pull request merged**